README.md

```diff
@@ -1 +1,109 @@
-# capstone-1
\ No newline at end of file
+<a name="readme-top">Fitness</a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 'Fitness <a name="about-project"></a>
+
+**Fitness Website** is a project based on first mobile structure and it is complete static website with even desktop responsive version.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```

Comment:
- **[OPTIONAL]** ⚠️ **IMPORTANT** ⚠️ Please 🙏🏽 add he LIVE DEMO section with the link of your app deployed with Github pages, it is mandatory for this capstone project.

![image](https://user-images.githubusercontent.com/58642949/234668753-8f60487f-4454-4600-80f0-bd86bf5caee9.png)